features/networking/ptp_operator.feature

```diff
@@ -0,0 +1,16 @@
+Feature: PTP related scenarios
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25940
+  @admin
+  Scenario: ptp operator can be deployed successfully
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-ptp" project
+    #check ptp related pods are ready in openshift-ptp projects
+    And I wait up to 120 seconds for the steps to pass:
```

Comment:
@huiran0826 you don't need to wrap it around the steps below already has time-out built-in

Reply:
@pruan-rht Thanks peter, remove the waiting step.